commit bd0fa644f8240d731ca435168d7863ce70f1bcae
Author: Oswald Buddenhagen <oswald.buddenhagen@digia.com>
Date:   Mon Nov 5 18:41:16 2012 +0100

    rebuild configure
    
    d7e922e6: QtNetwork: introduce configure switch to use system proxies by default
    343f559b: Support other platform names on configure.exe
    
    Change-Id: I2fc57d4ce36469bfe2b7c088375371d229bf8c7d
    Reviewed-by: Akseli Salovaara <akseli.salovaara@digia.com>
    Reviewed-by: Sergio Ahumada <sergio.ahumada@digia.com>

diff --git a/configure.exe b/configure.exe
index 6ffd4d9..9f57673 100755
Binary files a/configure.exe and b/configure.exe differ
